bin/riemann-consul

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env ruby
+
+# Reports consul services status to riemann
+
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+require 'socket'
+require 'net/http'
+require 'uri'
+require 'json'
+
+class Riemann::Tools::ConsulHealth
+  include Riemann::Tools
+
+  opt :consul_host, "Consul API Host (default to http://localhost:8500/)", :default => "localhost:8500/"
+
+  def initialize
+
+    @hostname = URI.parse(opts[:consul_host]).hostname
+    @consul_leader_url = URI.parse(opts[:consul_host] + "v1/status/leader")
+    @consul_services_url = URI.parse(opts[:consul_host] + "v1/catalog/services")
+    @consul_health_url_prefix = opts[:consul_host] + "v1/health/service/"
+
+  end
+
+  def alert(hostname, service, state, metric, description)
+
+    opts = {  :host => hostname,
+              :service => service.to_s,
+              :state => state.to_s,
+              :metric => metric,
+              :description => description }
+
+    report(opts)
+  end
+
+  def get(url)
+    Net::HTTP.get_response(url).body
+  end
+
+  def tick
+
+    leader = get(@consul_leader_url)
+    leader_hostname = URI.parse("http://" + leader[1..-2]).hostname
+
+    if (leader_hostname == @hostname)
+      services = JSON.parse(get(@consul_services_url))
+      for service in services
+        service_name = service[0]
```

Comment:
Hey, any chance you could make that change for me? Would love to get this merged. :)